js/src/api/util/identity.js

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import blockies from 'blockies';
+import blockies from 'ethereum-blockies';
```

Comment:
Interface is not 100% the same - on line 25 where we call `blockies({..})` the new interface is `blockies.create({..})` - not sure about the other parameters